lib/tasks/replace_upd_to_obj_upd_prohibited.rake

```diff
@@ -0,0 +1,42 @@
+require 'benchmark'
+
+namespace :locked_domains do
+
+  # Add new status instruction!
+  # First run `rake locked_domains:add_new_status`
+  # and then after finish first task run `rake locked_domains:remove_old_status`
+  desc 'Add serverObjUpdateProhibited for locked domains'
+  task add_new_status: :environment do
+    time = Benchmark.realtime do
+      ReplaceUpdToObjUpdProhibitedJob.perform_later('add')
```

Comment:
Consider using symbols instead of strings as arguments, preferably like `perform_later(action: :add)` or `(action: :remove)`